extensions/integration-tests/src/test/cms.test.ts

```diff
@@ -6,7 +6,7 @@
 import 'mocha';
 import * as vscode from 'vscode';
 import * as azdata from 'azdata';
-import * as mssql from '../../../mssql';
+import * as mssql from '../../../mssql/src/mssql';
```

Comment:
can you explain this change a bit

Reply:
This was just being more explicit with the path - but it doesn't need to change (I was changing other things and just didn't revert this). I'll change it back.